vagrant/environments/foss/config.yaml

```diff
@@ -55,7 +55,7 @@ nodes:
     forwarded_port:
       guest: 443
       host: 1943
-  - role: gitlab
+  - role: git
```

Comment:
This has impact on the relevant role , but it's ok, as soon as hieradata/role/git.yaml has what you need

Reply:
there is no role gitlab.yaml, only git.yaml. and git.yaml is working flawlessly out of the box.